.jpb/jpb-settings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Let's remove this file from the PR.

Reply:
yea not sure how that got there. Fixed it!